src/model_signing/_cli.py

```diff
@@ -88,6 +99,10 @@
 )
 
 
+def _collect_git_related_files(model_path: pathlib.Path) -> list[str]:
+    return glob.glob(join(model_path, ".git*"))
```

Comment:
This could be `model_path.glob("**/.git*")` to go into all directories or `model_path.glob(".git*")` to only use top-level

Reply:
Thanks, took the former.